packages/app/src/HDXMultiSeriesTimeChart.tsx

```diff
@@ -511,7 +511,7 @@ export const MemoChart = memo(function MemoChart({
         <Tooltip
           content={<HDXLineChartTooltip numberFormat={numberFormat} />}
           wrapperStyle={{
-            zIndex: 1000,
+            zIndex: 0.1,
```

Comment:
Seems like a good change, but I wonder why the 1000 value was there to begin with. We can merge and find out later :)

Reply:
Answer: https://linear.app/hyperdx/issue/HDX-1576/chart-legend-hidden-by-search-bar-header-in-search-page